src/scss/_layout.scss

```diff
@@ -0,0 +1,14 @@
+.l-container {
+  display: grid;
+  grid-template-columns: 150px 1fr;
+  min-height: calc(100vh - 48px);
+}
+
+.l-side {
+  background: #F7F7F7;
```

Comment:
Can't access a Vanilla variable here.. will address.

Reply:
Thanks for the note, are you going to do this in a quick follow-up? Or should we add an issue to follow-up?

Reply:
Raised issue: https://github.com/canonical-web-and-design/jaas-dashboard/issues/13